### PR TITLE
added workflow for autocommenting on PR Merged

### LINKED ISSUE #90: [feature] Auto Commenting feature for PR Merged
**Is your feature request related to a problem? Please describe.**
This feature aims to address the issue of inconsistent and delayed communication that occurs after merging pull requests. By automating the commenting process, it ensures that contributors receive timely feedback and acknowledgments. Additionally, it reduces the administrative burden on maintainers, allowing them to concentrate more on code reviews and other critical tasks.

### PR DESCRIPTION
**Pull request by Ratangulati**
Fixes #90

### CODE REVIEW

**Comment by Ratangulati** [comment]
@Soumya-Kushwaha I have added the workflow. Also please merge PR #131 
Thankyou

File: .github/workflows/autocomment_on_PR_merge.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    - name: Add Comment to Issue
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        COMMENT=$(cat <<EOF
+        {
+          "body": "🎉 Your pull request has been successfully merged! 🎉 Thank you for your valuable contribution to our project. Your efforts are greatly appreciated. Feel free to reach out if you have any more contributions or if there's anything else we can assist you with. Keep up the fantastic work! 🚀"
```
Comment (dinxsh) [inline .github/workflows/autocomment_on_PR_merge.yml:28]: add `contact mentors or soumya when needed` and mention them

**Review by dinxsh — CHANGES_REQUESTED** [review]
complete these

**Comment by Ratangulati** [comment]
Screenshot 2024-07-12 at 7 47 49 PM

Here is the updated output.

**Review by dinxsh — APPROVED** [review]
lgtm. great work!

**Comment by Ratangulati** [comment]
Please assign it the level and check my another PR #131

**Review by Soumya-Kushwaha — APPROVED** [review]

**Pull request merged**